scripts/bootstrap/buildbootstrapcli.sh

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+usage()
+{
+    echo "Builds a bootstrap CLI from sources"
+    echo "Usage: $0 [BuildType] --rid <Rid> --seedcli <SeedCli> [--arch <Architecture>] [--os <OS>] [--clang <Major.Minor>] [--corelib <CoreLib>] [--crossgen]"
+    echo ""
+    echo "Options:"
+    echo "  BuildType               Type of build (-debug, -release), default: -debug"
+    echo "  -arch <Architecture>    Architecture (x64, x86, arm, arm64, armel), default: x64"
+    echo "  -clang <Major.Minor>    Override of the version of clang compiler to use"
+    echo "  -config <Configuration> Build configuration (debug, release), default: debug"
+    echo "  -corelib <CoreLib>      Path to System.Private.CoreLib.dll, default: use the System.Private.CoreLib.dll from the seed CLI"
+    echo "  -os <OS>                Operating system (used for corefx build), default: Linux"
+    echo "  -rid <Rid>              Runtime identifier (without the architecture part)"
+    echo "  -seedcli <SeedCli>      Seed CLI used to generate the target CLI"
+}
```

Comment:
Consider adding an `outputpath` option

Reply:
Do you mean a path that would override the default path which is the <Rid> subdirectory of the current directory or did you mean path for just the generated bootstrap CLI?

Reply:
I"m thinking for the generated bootstrap CLI